src/StatsBase.jl

```diff
@@ -35,6 +35,7 @@ module StatsBase
     wmean!,             # weighted mean across dimensions with provided storage
     wmedian,            # weighted median
     wquantile,          # weighted quantile
+	wiqr,				# weighted IQR
```

Comment:
Hmm... The indentation looks fine on mine computer. I wonder why it looks funky here?

Reply:
That's because the character is a tab. Change it to normal spaces.